whois-commons/src/main/java/net/ripe/db/whois/common/DatabaseDummifierJmx.java

```diff
@@ -64,6 +66,16 @@ public class DatabaseDummifierJmx extends JmxBase {
     private static final AtomicInteger jobsAdded = new AtomicInteger();
     private static final AtomicInteger jobsDone = new AtomicInteger();
 
+    private enum Environment {
+        DEV,
+        PREPDEV,
```

Comment:
We use environment property in sentry too, can you shift to a separate  Enum class so we can use this enum there as well?

Reply:
Done